src/Symfony/Bundle/FrameworkBundle/Tests/Controller/ControllerNameParserTest.php

```diff
@@ -108,7 +108,6 @@ public function getMissingControllersTest()
     }
 
     /**
-     * @expectedException
```

Comment:
This test won't fail if there was no exception thrown. A solution might be adding $this->fail() call after calling $parser->parse($bundleName);.